app/src/main/res/layout/fragment_example2.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:gravity="center"
     android:paddingBottom="@dimen/activity_vertical_margin"
     android:paddingLeft="@dimen/activity_horizontal_margin"
     android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
-    tools:context="io.intrepid.skeleton.MainActivity"
     >
 
     <TextView
+        android:id="@+id/example2_text"
```

Comment:
Maybe add some styles for Text so that we are following the guidelines for typography and color in android 

text sizing spec -> (https://material.google.com/style/typography.html#typography-styles)

e.g.(https://github.com/IntrepidPursuits/MITMobius-android/blob/develop/app/src/main/res/values/styles/values/styles_text.xml)